### DISCUSSION TITLE
Is it possible to add rainbow color support

### DISCUSSION THREAD

**Discussion by LansV** [hizzgdev/jsmind#539] [General]
Is it possible to add rainbow color support

**Comment by hizzgdev** [comment]
what does `rainbow color` mean?

jsMind supports setting the background color for nodes, so you can give different background colors to different nodes.
jsMind supports setting the background image for nodes, so you can give a rainbow background image to nodes.

doc: https://github.com/hizzgdev/jsmind/blob/master/docs/en/3.operation.md#35-setting-style
